en/editor/publish/publish-bytedance-mini-game.md

```diff
@@ -55,6 +55,25 @@ Specifically, developers need to do the following:
 4. Delete the **res** folder inside the local release package.
 5. For the test phase, you may not be able to deploy to the official server, you need to use the local server to test, then open the details page in the WeChat DevTools, check the `Does not verify valid domain names, web-view (business domain names), TLS versions and HTTPS certificates` option in the **Local Settings**.
 
+## Subpackage Loading
+
+The ByteDance Mini Game requires a specific version to support the subpackage feature, and the version requirements for the ByteDance product are as follows:
+
+| product | Android     | iOS        |
+| :--     | :---        | :---       |
+| TikTok  | v13.6.0     | v13.7.0    |
+| TouTiao | v7.9.9      | v7.9.8     |
+
+The ByteDance Dev Tool requires a version greater than or equal to **2.0.6** and less than **3.0.0**. The debugging base library requires 1.88.0 or above.
```

Comment:
For ByteDance DevTools, please use **v2.0.6** and above, but below **v3.0.0**. After updating the DevTools, don't forget to modify the **Debug Base library** to **1.88.0** and above in the ByteDance DevTools.